topology/University of Wisconsin/CHTC/CHTC.yaml

```diff
@@ -502,6 +508,30 @@ Resources:
     VOOwnership:
       GLOW: 100
 
+  CHTC-ap40:
+    Active: true
+    Description: The ap40.chtc.wisc.edu login node for the OSG Open Pool
+    ID: 1468
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Aaron Moate
+          ID: OSG1000015
+      Security Contact:
+        Primary:
+          Name: Aaron Moate
+          ID: OSG1000015
+    FQDN: ap40.chtc.wisc.edu
+    Services:
+      Submit Node:
+        Description: OS Pool access point
+        Details:
+          hidden: false
+    Tags:
+      - OSPool
+    VOOwnership:
+      GLOW: 100
```

Comment:
```suggestion
      OSG: 100
```
This is an access point entirely for the OSPool so listing it as owned by the "OSG" VO is more accurate.